.github/workflows/tests.yml

```diff
@@ -144,6 +146,9 @@ jobs:
         with:
           name: cruby-gem
           path: gems
+      - name: Update pkgconf
+        if: env.MATRIX_RUNS_ON == 'windows-2019' || env.MATRIX_RUNS_ON == 'windows-2022'
+        run: pacman -Sy --noconfirm pkgconf mingw64/mingw-w64-x86_64-pkgconf
```

Comment:
I was puzzled at first why https://github.com/mudge/re2/actions/runs/7039787062/job/19159476634#step:6:190 was failing. Adding debugging showed the wrong version of `pkgconf` was installed:

```
=== pkg-config version: 2.0.3
```

Then I noticed https://github.com/mudge/re2/actions/runs/7039660736/job/19159114189 passed successfully since 2.1.0 was installed, and then realized `mingw64/mingw-w64-x86_64-pkgconf` was the actual binary needed by Windows here.

Reply:
I’m unfamiliar with `pacman` but are you installing two packages here? One called [`pkgconf`](https://packages.msys2.org/package/pkgconf) and one called [`mingw64/mingw-w64-x86_64-pkgconf`](https://packages.msys2.org/package/mingw-w64-x86_64-pkgconf?repo=mingw64)? What’s the difference? Is one a binary vs one that compiles from source?

Reply:
Yes, there are two packages. Likely we only need the `mingw64` version, but I included the other one just in case. https://www.msys2.org/wiki/MSYS2-introduction/ has an explanation:

> MSYS2 consists of three subsystems and their corresponding package repositories, msys2, mingw32, and mingw64.
>
> The mingw subsystems provide native Windows programs and are the main focus of the project. These programs are built to co-operate well with other Windows programs, independently of the other subsystems. This part builds on the MinGW-w64 project.
>
> The msys2 subsystem provides an emulated mostly-POSIX-compliant environment for building software, package management, and shell scripting. These programs live in a virtual single-root filesystem (the root is the MSYS2 installation directory). Some effort is made to have the programs work well with native Windows programs, but it's not seamless. This part builds on the Cygwin project.

Reply:
I just dropped the `pkgconf` entry since abseil doesn't build with Cygwin.

Reply:
For some reason, Windows 2022 builds don't appear to have `pkgconf` installed (https://github.com/mudge/re2/actions/runs/7052548164/job/19197882051?pr=118), unlike Windows 2019 (https://github.com/mudge/re2/actions/runs/7052548164/job/19197881418?pr=118). 

My only guess that Windows 2022 ships with the standard `pkg-config`, which doesn't have the bugs in `pkgconf`. https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md doesn't really show much difference from https://github.com/actions/runner-images/blob/main/images/windows/Windows2019-Readme.md, though.

Reply:
Does this mean we don’t need it for windows-2022 builds or do we depend on pkgconf (rather than pkg-config) for Windows?

Reply:
I'm not sure. I wish there were a way to inspect the GitHub VMs because it's really opaque to me what's actually installed. https://github.com/msys2/MSYS2-packages/issues/2872 claims that you can't install `pkgconf` if `pkg-config` is already installed. I'll try to drop it from Windows 2022 and see what happens.

Reply:
It looks like there was an update to the Windows 2019 image in the past 24 hours: https://github.com/actions/runner-images/commit/45a5b970b809e08869dcf5f11456001d86952d3f

Reply:
Yeah, the latest builds seem to work fine without the `pkgconf` upgrade now. I added a `pkg-config --version` check to make it easier to triage if the problem happens again.